tests/MongoDBDriverManager_debugInfo.phpt

```diff
@@ -34,7 +34,7 @@ array(2) {
       ["is_passive"]=>
       bool(false)
       ["last_is_master"]=>
-      array(8) {
+      array(9) {
```

Comment:
Would it make more sense to just use %d here, as we typically do for object fields (even when we enumerate all of them in the expected output)?